src/main/java/com/squareup/javapoet/MethodSpec.java

```diff
@@ -63,6 +63,8 @@ private MethodSpec(Builder builder) {
         "last parameter of varargs method %s must be an array", builder.name);
 
     this.name = checkNotNull(builder.name, "name == null");
+    // Add parameter Javadoc to the method Javadoc
+    builder.javadoc.add(builder.parameterJavadoc.build());
```

Comment:
I don't think this is correct. If you build a method and then call toBuilder(), add javadoc, and add a parameter you'll get a split set of `@param`. We need to track the Javadoc tags separately from the original Javadoc.